src/routes/LogIn/LogIn.jsx

```diff
@@ -41,31 +44,32 @@ export default function LogIn() {
   }
 
   return (
-    <div className={styles.bobyContainer}>
+    <div className={styles.bodyContainer}>
+      <button className={styles.registerBtn} onClick={()=>{ navigate('/register') }}>가입하기</button>
       <div className={styles.card}>
         <div className={styles.cardContainer}>
-          <p>Log in</p>
+          <p>로그인</p>
 
           <div className={styles.inputWrap}>
-            <p> ID</p>
+            <p>아이디</p>
             <FaceOutlinedIcon className={styles.inputIcon} />
-            <input type="text" onChange={e => setId(e.target.value)} className={cx(styles.input, {[styles.inputBorderActive]: id}) } />
-            { invailidMsg.indexOf('ID') > -1 && <p className={styles.inputTextUnder}> {invailidMsg} </p> }
+            <input type="text" onChange={e => setId(e.target.value)} className={cx(styles.logInInput, { [styles.inputBorderActive]: id })} />
```

Comment:
```suggestion
            <input type="text" onChange={e => setId(e.currentTarget.value)} className={cx(styles.logInInput, { [styles.inputBorderActive]: id })} />
```

target과 currentTarget을 구분해야 할텐데, 항상 currentTarget만 쓰는게 좋습니다.

Reply:
감사합니다. target과 currentTarget 차이점 숙지 후 적용하도록 하겠습니다.